crates/router/src/core/payments/helpers.rs

```diff
@@ -5265,3 +5274,26 @@ pub async fn validate_merchant_connector_ids_in_connector_mandate_details(
     }
     Ok(())
 }
+
+pub fn is_cobadged_card(card_number: CardNumber, card_cvc: masking::Secret<String>) -> bool {
```

Comment:
can we please move this to impl of Card domain model

Reply:
Moved the `is_cobadged_card()` into impl of CardNumber struct